docs/user_guide/data_scientist_guide/federated_xgboost/secure_xgboost_user_guide.rst

```diff
@@ -110,15 +132,59 @@ As shown above, histogram-based XGBoost in horizontal and vertical collaboration
      - ❌
      - ❌
 
-Implementation Notes:
+Note on Implementation:
 ---------------------
 
 - **Horizontal mode**:
-  - Histogram-based horizontal model does not need GPU encryption, as it is not as computationally intensive (encrypt histogram vectors) as in vertical mode (encrypt gradients).
+  - The histogram-based horizontal model does not need GPU encryption, as it is not as computationally intensive (encrypting histogram vectors) as vertical mode (encrypting gradients).
 - **Vertical mode**:
-  - Primary goal (gradient protection) is fully supported across all combinations
-  - Secondary goal (split value masking) is only supported with CPU XGBoost, regardless of encryption type
+  - Primary goal (gradient protection) is fully supported across all combinations.
+  - Secondary goal (split value masking) is only supported with CPU XGBoost, regardless of encryption type.
+
+Advanced Topics: Future Security Scenarios
+===========================================
+
+The following security scenarios are not currently implemented in our solution. Users should be aware that **plaintext histogram communication can reveal data distribution information**, which may enable data reconstruction attacks.
+
+Potential Future Enhancements
+------------------------------
+
+.. list-table:: Future Security Scenarios
+   :widths: 15 12 15 20 25 20
+   :header-rows: 1
+
+   * - Collaboration Mode
+     - Algorithm
+     - Security Risk
+     - Trust Model
+     - Possible Approach
+     - Challenges
+   * - **Horizontal**
+     - Histogram-based
+     - Histogram leakage
+     - Trust server, no trust in clients
+     - Server performs calculations; distributes only final splits
+     - Rare trust assumption; uncommon in practice
+   * - **Horizontal**
+     - Histogram-based
+     - Histogram leakage
+     - No trust in server or clients
+     - Confidential computing
+     - HE compatibility issue [*]_
+   * - **Vertical**
+     - Histogram-based
+     - Histogram leakage
+     - Trust passive parties, no trust in active party
```

Comment:
similarly, assume passive party is already secured with one of PETs, not trust in active party